mautic/mautic
# Render help text for form fields

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:


This PR provides rendering the help message option for from fields.  A call to the [form_help](https://symfony.com/doc/current/form/form_customization.html#reference-forms-twig-help) function for the form_row and form_start blocks are added.
To use it, the `help` option needs to be added to form field.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.This PR is difficult to test because the `help` option must be added to the form fields. Therefore, a code review is needed.
